frontend/src/features/admin-form/create/builder-and-design/useCreateTabForm.ts

```diff
@@ -0,0 +1,15 @@
+import { useAdminForm } from '~features/admin-form/common/queries'
+
+import {
+  BuildFieldState,
+  stateDataSelector,
+  useBuilderAndDesignStore,
+} from './useBuilderAndDesignStore'
+
+export const useCreateTabForm = () => {
+  const stateData = useBuilderAndDesignStore(stateDataSelector)
+  return useAdminForm({
+    // Only fetch data if no field is being created or edited
+    enabled: stateData.state === BuildFieldState.Inactive,
```

Comment:
can be changed to 
```ts
enabled: stateData === null
```
if we used null to represent inactivity instead.

Reply:
keeping state management as-is as per previous comment